_maps/map_files/Pahrump/Pahrump-Surface-2.dmm

```diff
@@ -28079,6 +28835,31 @@
 	icon_state = "floorrusty"
 	},
 /area/f13/building)
+"lBD" = (
+/obj/item/lock_construct,
+/obj/item/lock_construct,
+/obj/item/key,
+/obj/item/key,
+/obj/structure/closet/cardboard{
```

Comment:
![image](https://user-images.githubusercontent.com/49745305/219883153-2bfcb26a-decf-4363-b014-2fcef3bd10ca.png)


Get these two boxes out of the way

Reply:
alright, I'll get on it, was hoping for a cluttered "moved in" feel for when the bartender unpacks his things, but I can make it clean off the bat.

Reply:
![image](https://user-images.githubusercontent.com/72716882/219900210-892010ab-e7c6-4bc4-8368-f66480f1a8c3.png)
Replaced with cobwebs, easier to clear out since you don't gotta find a spot to put the stuff.